content/docs/command-reference/plots/index.md

```diff
@@ -34,10 +34,10 @@ experiments. Usual plot examples are AUC curves, loss functions, confusion
 matrices, among others.
 
 This type of metrics files are created by users, or generated by user data
-processing code, and get defined with the `-p` (`--plots`) and
-`--plots-no-cache`) options of `dvc run`. `dvc plots` subcommands can work with
-plots files committed to a Git repo history, data files controlled by DVC, or
-any other file in system.
+processing code, and can be defined in `dvc.yaml` stages (using the `--plots`
+and `--plots-no-cache` options if using `dvc run`). `dvc plots show` and
+`dvc plots diff` can work with any valid plots files in the system, whether
+tracked by Git or DVC, or not.
```

Comment:
Fixes #1800 @pared @shcheklein but note that since `plots modify` does require that plots are defined in dvc.yaml, I couldn't change this to be as general as suggested initially (summarized in https://github.com/iterative/dvc.org/pull/1809#issuecomment-708029242):

> if explain in the plots concept itself that plots are not only dvc.yaml based people won't expect dvc.yaml to be present in the first place.

Reply:
I like the shift from `get defined` to `can be defined`. This is on point what we actually did.

> can work with any valid plots files in the system

While it is something we need to do (as part of iterative/dvc#4446, currently we cannot `dvc plots show {target}` in no-repo case, as we will get error that `'.' is not a git repo`.

Reply:
Created #4761 to address this

Reply:
OK great, thanks for addressing that. In that case we can leave this text as-is, I think.

Reply:
I think we are still overcomplicating this. In this doc the most important points to say that users can output certain type of metrics (arrays, etc) into JSON, YAML or whatnot and DVC provides a bunch of commands to deal with them - visualize, compare, etc, etc. We should def mention that it is similar to metrics, but for "continuous", etc, etc

Details that certain commands require `dvc.yaml` to be present can be hidden into those commands. Or there should be a separate section that starts with some _motivation_ for creating a `dvc.yaml`- what benefits on top of regular files people can get from using it.

Reply:
Well, I was following the original idea to emphasize that plots don't have to be tracked by DVC... But you're right, this was too low level for an index. I just removed a bunch of redundant info and left the basics + some refs to where the details are. Also updated show and diff a bit again. See 27519b4a. Resolving here